lib/screens/v2/candidate_generator/widgets/elevator_closures.ex

```diff
@@ -19,8 +19,9 @@ defmodule Screens.V2.CandidateGenerator.Widgets.ElevatorClosures do
     with {:ok, stop_sequences} <-
            RoutePattern.fetch_stop_sequences_through_stop(parent_station_id),
          {:ok, parent_station_map} <- Stop.fetch_parent_station_name_map(),
-         {:ok, elevator_closures, facility_id_to_name} <- fetch_elevator_closures(),
-         icon_map <- get_icon_map(elevator_closures) do
+         {:ok, elevator_closures, facility_id_to_name} <- fetch_elevator_closures() do
+      icon_map = get_icon_map(elevator_closures, parent_station_id)
```

Comment:
No real advantage. It's just that normally the `foo <- bar` lines should be things that you expect might fail. Basically it tries to match the left-hand side of the `<-`, and if any match fails it jumps over to the `else` block and tries matching the failed value against patterns in there instead.

Since `icon_map = get_icon_map(elevator_closures, parent_station_id)` will always match, it doesn't need to be in that special part of the `with` construct.